### ISSUE TITLE
TRANSACTIONS - Check Consumption Consistency

### ISSUE THREAD

**Issue by ClaudeROSSI** [sap-labs-france/ev-server#3506]
To be on the safe side and avoid billing inconsistent data we need to make sure the meter values  we receive are consistent. 

This does not occur so often. So the first approach will be to just detect and ignore (or cap) inconsistent values and log an erreur.
In the future, if this happens too often we will mark the session as being "IN error" and send a mail to the admins.

So far, the pb is only visible on a particular charging station:

![2022-05-03_08-28-20.png](https://images.zenhubusercontent.com/5fa82ce7552e0aed8eb43377/61e3b8e1-3cf3-4fe9-a977-9b58f2e4c01e)

**Comment by ClaudeROSSI** [comment]
@OliveGerste : Let me know if you can take this one.
Thanks,
Claude

**Comment by LucasBrazi06** [comment]
@ClaudeROSSI Maybe another developper like Mahdi would be better.

**Comment by mytrkr0** [comment]
What is the reason of this Mr. Claude Could you explain it? @ClaudeROSSI

**Comment by ClaudeROSSI** [comment]
> What is the reason of this Mr. Claude Could you explain it? @ClaudeROSSI

We faced the situation where a charging station sends inconsistent meter values. We need to detect these inconsistent values and get rid of them.

**Comment by mytrkr0** [comment]
Yes I got it. But I mean is what is the reason? Could you find it?
